articles/cognitive-services/Computer-vision/computer-vision-how-to-install-containers.md

```diff
@@ -82,7 +82,7 @@ The container image for the Recognize Text container is available from a private
 
 Use the [docker pull](https://docs.docker.com/engine/reference/commandline/pull/) command to download a container image from the repository. For example, to download the latest Recognize Text container image from the repository, use the following command:
 
-```Docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands